drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -584,16 +586,44 @@ def extract_sources(img, dqmask=None, fwhm=3.0, kernel=None, photmode=None,
         imgarr = img
 
     segm = detect_sources(imgarr, segment_threshold, npixels=source_box,
-                          filter_kernel=kernel)
+                          filter_kernel=kernel, connectivity=4)
+
+    log.debug("Creating segmentation map for {} ".format(outroot, kernel.shape))
+    if kernel is not None:
```

Comment:
Do you really intend to print both `outroot` and `kernel.shape` here?

Reply:
Corrected in 219bfa6.